backend/mensa_recommend/source/computations/user_location.py

```diff
@@ -1,96 +1,212 @@
-from datetime import datetime, date
+from datetime import datetime, date, time
+from mensa_recommend.source.computations.transformer import transform_week_day_to_int
```

Comment:
```suggestion
from mensa_recommend.source.computations.transformer import transform_week_day_to_int
from typing import Optional
```
Edited with smartphone 👀